asctb-api/src/routes/google-sheet.ts

```diff
@@ -28,11 +28,12 @@ export function setupGoogleSheetRoutes(app: Express): void {
         );
       }
       const data = papa.parse(response.data).data;
-
+      const generalPublications = await fetchGeneralPublicationsData(data);
       const asctbData = await makeASCTBData(data);
 
       return res.send({
         data: asctbData,
+        generalPublications: generalPublications,
```

Comment:
Totally wrong approach. You should be returning a set of metadata dictionaries here with key/value pairs pulled form the metadata section (keys can be mapped for consistency) of the ASCT+B tables (note plural as there is a feature for sending more than one table at a time). Also, graph, json-ld, and OWL responses will need updated to include this data as well. I may have some time between 10 and 11 this morning to go over this with you if you like.

Reply:
Sure Bruce, we can discuss this after daily stand up meeting.